easybuild/easyconfigs/c/CUDA/CUDA-7.0.28.eb

```diff
@@ -0,0 +1,18 @@
+name = 'CUDA'
+version = '7.0.28'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://developer.download.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/']
+
+sources = ['%(namelower)s_%(version)s_linux.run']
+
+modextravars = {'CUDA_HOME': '%(installdir)s'}
```

Comment:
@thiell the CUDA easyblock already defines this, no need to include it here

(there was a bug fix in EasyBuild v2.5.0 related to this, cfr. https://github.com/hpcugent/easybuild-framework/pull/1519)